### PR TITLE
docs: add Homebrew information for Nerd Fonts

### PR DESCRIPTION
**Pull request by caitlynrw**
### Prerequisites

- [X] I have read and understood the [contributing guide][CONTRIBUTING.md]
- [x] The commit message follows the [conventional commits][cc] guidelines
- N/A Tests for the changes have been added (for bug fixes/features)
- N/A Docs have been added/updated (for bug fixes/features)

### Description
Adds information about using Homebrew to install Nerd Fonts

### CODE REVIEW

File: website/docs/installation/fonts.mdx [hunk 1]
```diff
@@ -23,6 +23,15 @@ This command needs to be executed as admin. The fonts are installed system wide.
 oh-my-posh font install
 ```
 
+If you're using Homebrew, you can tap the `homebrew-cask-fonts` repository and install Nerd Fonts from there instead:
```
Comment (JanDeDobbeleer) [inline website/docs/installation/fonts.mdx:41]: Can we use `Tabs` here to allow the use to switch between the methods? Just like we do in the Installation and shell selection?

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: website/docs/installation/fonts.mdx (hunk 1 above)
Comment (caitlynrw) [reply website/docs/installation/fonts.mdx:41]: Should be implemented now.

File: website/docs/installation/fonts.mdx [hunk 2]
```diff
@@ -13,6 +16,15 @@ To see the icons displayed in Oh My Posh, **install** a [Nerd Font][nerdfonts],
 
 #### Installation
 
+<Tabs
+    defaultValue="oh-my-posh"
+    values={[
+        { label: 'oh-my-posh', value: 'oh-my-posh' },
```
Comment (JanDeDobbeleer) [inline website/docs/installation/fonts.mdx]:
```suggestion
        { label: 'Oh My Posh', value: 'oh-my-posh' },
```

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: website/docs/installation/fonts.mdx (hunk 2 above)
Comment (caitlynrw) [reply website/docs/installation/fonts.mdx]: Shouldn't the `label` be the one changed?

Comment (JanDeDobbeleer) [reply website/docs/installation/fonts.mdx]: You are correct!

**Comment by JanDeDobbeleer** [comment]
@DarkMagicSource sorry for the delay, I fixed the issues and this is ready to merge.

**Comment by JanDeDobbeleer** [comment]
@all-contributors please add @DarkMagicSource for doc

**Pull request merged**